src/app/code/community/IntegerNet/Solr/etc/config.xml

```diff
@@ -39,6 +39,12 @@
                     <layer_filter_category>IntegerNet_Solr_Model_Catalog_Layer_Filter_Category</layer_filter_category>
                 </rewrite>
             </catalog>
+
+            <iways_paypalplus>
```

Comment:
No, in the PayPalPlus module the autoloader is a model. I decided to but the rewrite into "helpers" to have it close to our own autoloader